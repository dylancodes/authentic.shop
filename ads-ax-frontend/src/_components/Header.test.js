import React from 'react';
import { cleanup, render, fireEvent } from 'react-testing-library';
import 'jest-dom/extend-expect';
import { BrowserRouter as Router } from 'react-router-dom';

import Screen from './Header.js';

describe('The Header', () => {
  test('should match and pass the screenshot test', () => {
    const { container } = render(
      <Router>
        <Screen />
      </Router>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
