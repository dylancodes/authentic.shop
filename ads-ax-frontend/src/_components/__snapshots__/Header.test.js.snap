// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header should match and pass the screenshot test 1`] = `
<div
  class="adbc--header__div"
>
  <a
    href="/dashboard"
  >
    <h1>
      Admin.Authentic.Shop
    </h1>
  </a>
</div>
`;
